src/api/window/windowOption.ts

```diff
@@ -46,3 +46,10 @@ export interface CustomRequestHeaders {
     urlPatterns: Array<string>;
     headers: Array<object>;
 }
+
+export type WindowOptionDiff = {
```

Comment:
Looks like this has a bit more changes than intended.

Reply:
Sorry about that... fixed.